File: 0920-uncommon-words-from-two-sentences/0920-uncommon-words-from-two-sentences.java
class Solution {
    public String[] uncommonFromSentences(String s1, String s2) {
        Map<String, Integer> cnt = new HashMap();
        for(String word : s1.split(" ")) cnt.put(word, cnt.getOrDefault(word, 0)+1);
        for(String word : s2.split(" ")) cnt.put(word, cnt.getOrDefault(word, 0)+1);
        List<String> ans = new LinkedList();
        for(String word : cnt.keySet()){
            if(cnt.get(word) == 1) ans.add(word);
        }
        return ans.toArray(new String[ans.size()]);
    }
}

File: 0920-uncommon-words-from-two-sentences/0920-uncommon-words-from-two-sentences.cpp
class Solution {
public:
    vector<string> uncommonFromSentences(string s1, string s2) {
        stringstream str1(s1), str2(s2);
        unordered_map<string, int> cnt;
        string word;
        while(str1 >> word) cnt[word]++;
        while(str2 >> word) cnt[word]++;
        vector<string> ans;
        for(auto x : cnt){
            if(x.second == 1) ans.push_back(x.first);
        }
        return ans;
    }
};